// const error =  new Error("This is a error")
// // console.log(error.stack)
// // console.log(error.message)
// class Custom_error extends Error
// {
//     constructor(message)
//     {
//         super(message);
//     }
// }
// throw new Custom_error("The custom error has happened")